validate_ot_nexson.py: WARNING: No trees were found, or all trees were flagged for deletion in "trees(id=trees2)"
